wondrous-app/components/Common/Sidebar/Settings/index.tsx

```diff
@@ -0,0 +1,285 @@
+    label: 'General Settings',
+    value: 'general',
+    href: orgId ? `/organization/settings/${orgId}/general` : `/pod/settings/${podId}/general`,
+    page: [SettingsPage.Org, SettingsPage.Pod],
+  },
+  {
+    Icon: WrenchIcon,
+    label: 'Configure Wallet',
+    value: 'wallet',
+    href: orgId ? `/organization/settings/${orgId}/wallet` : `/pod/settings/${podId}/wallet`,
+    page: [SettingsPage.Org, SettingsPage.Pod],
+  },
+  {
+    Icon: LockIconOutline,
+    label: 'Token Gating',
+    value: 'token-gating',
+    href: `/organization/settings/${orgId}/token-gating`,
+    page: [SettingsPage.Org],
+  },
+  {
+    Icon: HexagonIcon, // need icon
+    label: 'Integrations Settings',
+    value: 'integrations',
+    href: orgId ? `/organization/settings/${orgId}/integrations` : `/pod/settings/${podId}/integrations`,
+    page: [SettingsPage.Org],
+  },
+  {
+    Icon: ReceiptIcon,
+    label: 'Payments Ledger',
+    value: 'payouts',
+    href: orgId ? `/organization/settings/${orgId}/payouts` : `/pod/settings/${podId}/payouts`,
+    page: [SettingsPage.Org, SettingsPage.Pod],
+  },
+  {
+    Icon: ReceiptIcon,
+    label: 'Payment Method',
+    value: 'payment-method',
+    href: `/organization/settings/${orgId}/payment-method`,
+    page: [SettingsPage.Org],
+  },
+  {
+    Icon: GroupIcon,
+    label: 'Members',
+    value: 'members',
+    href: orgId ? `/organization/settings/${orgId}/members` : `/pod/settings/${podId}/members`,
+    page: [SettingsPage.Org, SettingsPage.Pod],
+  },
+  {
+    Icon: RolesIcon,
+    label: 'Roles',
+    value: 'roles',
+    href: orgId ? `/organization/settings/${orgId}/roles` : `/pod/settings/${podId}/roles`,
+    page: [SettingsPage.Org, SettingsPage.Pod],
+  },
+  {
+    Icon: NotificationsIcon,
+    label: 'Notifications',
+    value: 'notifications',
+    href: orgId ? `/organization/settings/${orgId}/notifications` : `/pod/settings/${podId}/notifications`,
+    page: [SettingsPage.Org, SettingsPage.Pod],
+  },
+  {
+    Icon: FileDownloadIcon,
+    label: 'Task Import',
+    value: 'import',
+    href: `/organization/settings/${orgId}/task-import`,
+    page: [SettingsPage.Org],
+  },
+  {
+    Icon: NotificationsIcon,
+    label: 'Notifications',
+    value: 'notifications',
+    href: `/profile/notifications`,
+    page: [SettingsPage.Profile],
+  },
+  {
+    Icon: () => (
+      <GitHubIcon
+        style={{
+          color: '#525252',
+        }}
+      />
+    ),
+    label: 'Github',
+    value: 'github',
+    href: orgId ? `/organization/settings/${orgId}/github` : `/pod/settings/${podId}/github`,
+    page: [SettingsPage.Pod],
+  },
+];
+
+function SettingsWrapper(props) {
+  const { children, showPodIcon = true } = props;
+
+  const router = useRouter();
+  const user = useMe();
+  const { minimized } = useSideBar();
+
+  const { pathname } = router;
+  const { orgId, podId } = router.query;
+  const { data: userPermissionsContext } = useQuery(GET_USER_PERMISSION_CONTEXT, {
```

Comment:
latest staging has this fetch done at layout level, I think we can reuse that context ( CreateEntityContext )